client/src/app/components/employee-settings/employee-settings.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="edit-container">
+  <button mat-button (click)="editMode(!edit)">{{edit ? "Cancel": "Edit"}}</button>
+
+  <div *ngIf="employee">
+    <mat-card>
+      <mat-card-header>
+        <mat-card-title>
+          {{edit? "Update": ""}} {{employee?.first_name + " " + employee?.last_name}}
+        </mat-card-title>
+      </mat-card-header>
+
+      <mat-card-content>
+
+          <mat-form-field>
+            <mat-label for="First Name"></mat-label>
+            <input matInput id="first_name" type="text" [disabled]="true" [value]="employee.first_name"/>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label for="Last Name"></mat-label>
+            <input matInput id="last_name" type="text" [disabled]="true" [value]="employee.last_name"/>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label for="email">Email</mat-label>
+            <input matInput id="email" type="text" [(ngModel)]="employee.email" email [disabled]="!edit"/>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label for="password">Password</mat-label>
+            <input matInput id="password" type="password" [(ngModel)]="password" minlength="5" [disabled]="!edit"/>
```

Comment:
I think the password should be moved to another form just under it with 2 inputs (new password, confirm password), and this will be updated through a specific endpoint that is just responsible for updating the password

Reply:
So right now, it's actually not associated to any form.
This was something that I wanted to discuss as well, because for the selection inputs (role and active), I couldn't find a viable way to set the default values on those fields. Using ngModel has been a cleaner alternative, but the consideration we have to make is how we validate the input.
This is something we can always refactor later if we want to add validation, but just some food for thought for now.